README.md

```diff
@@ -2,6 +2,7 @@
 
 [![License](https://img.shields.io/github/license/RevenueCat/purchases-android.svg?style=flat)](https://github.com/RevenueCat/purchases-android/blob/main/LICENSE)
 [![Release](https://img.shields.io/github/release/RevenueCat/purchases-android.svg?style=flat)](https://github.com/RevenueCat/purchases-android/releases)
+[![Coverage Status](https://coveralls.io/repos/github/RevenueCat/purchases-android/badge.svg?branch=main)](https://coveralls.io/github/RevenueCat/purchases-android?branch=main)
```

Comment:
Hi again @mikescamell!

Thanks again for your contribution! We like the PR overall but we prefer not to add this badge at this time. The reasons for this are that we think that coverage is not by itself a good indicator of the codebase robustness and you might be able to achieve great coverage whilst still not having a robust set of tests. For example, since coverage only checks that lines are called within tests, you could have a suite of tests that call all methods while not accurately checking the values and logic are correct. Because of this, it can be a misleading metric. We choose not to focus on coverage by itself but rather on the quality of our tests, and we fear having a badge for coverage might lead us into focusing on the percentage rather than quality. People can still look at the coverage in [codecov](https://app.codecov.io/gh/RevenueCat/purchases-android) directly, which is open to everyone.

Could you make this change? Then we should be good to merge this.

Thanks again!

Reply:
Totally agree! Best metric around code coverage is to have something > 0%, that means you're adding tests and on the right path 😅 

Quoting Martin Fowler:

> The key is to test the areas that you are most worried about going wrong. That way you get the most benefit for your testing effort.